api/lib/domain/usecases/create-organization-invitation.js

```diff
@@ -11,18 +12,37 @@ async function _checkMemberNotExistWithOrganizationIdAndEmail(membershipReposito
   }
 }
 
-async function _checkOrganizationInvitationNotExistsWithOrganizationIdAndEmail(organizationInvitationRepository, organizationId, email) {
-  const organizationInvitationsFound = await organizationInvitationRepository.findByOrganizationIdAndEmail({ organizationId, email });
-  if (organizationInvitationsFound.length) {
-    throw new AlreadyExistingOrganizationInvitationError(`Invitation already exists with the organization id ${organizationId} and the email ${email}`);
-  }
+// TODO export to a key/code service
+function _generateTemporaryKey() {
+  return Math.random().toString(36).substring(2).toUpperCase();
```

Comment:
Excellent, me gusta

Reply:
Noooon ce n’est pas une bonne technique de génération de clé aléatoire ! Cf. par exemple https://stackoverflow.com/questions/5651789/is-math-random-cryptographically-secure

Reply:
Ok.
Je vais utiliser randomString comme dans campaign-code-generator.js

Reply:
Done